ERROR_MESSAGES.md

```diff
@@ -7,7 +7,7 @@ Any text in the following format `(Example)` are considered variables to be fill
 * 2 : The file is not a valid XML document
 * 3 : Unexpected exception occurred during conversion
 * 4 : Unexpected exception occured during encoding
-* 5 : The file is not a QRDA-III XML document. Please ensure that the submission complies with the most recent implementation guide. https://ecqi.healthit.gov/system/files/eCQM_QRDA_EC-508_0.pdf#page=19
+* 5 : The file is not a QRDA-III XML document. Please ensure that the submission complies with the `(Submission year's)` implementation guide. https://ecqi.healthit.gov/system/files/eCQM_QRDA_EC-508_0.pdf#page=19
```

Comment:
What about making the URL a field that gets filled in like the submission year?  That way, when the submission year get's updated, hopefully the URL will get updated instead of being forgotten in this error message.

Reply:
k

Reply:
Agreed